src/FundraiserMailingGenerator/locales/en.js

```diff
@@ -15,6 +15,6 @@ export const amounts = [];
 export const donorButtonDefaults = `Donate {{amount}} now`;
 export const donorLinkDefaults = `Donate another amount`;
 export const nonDonorLinkDefaults = `Will you chip in {{amount}} to help [EDIT]?`;
-export const oneClickCopy = `If you’ve saved your payment information with SumOfUs, your donation will go through immediately:`;
+export const oneClickCopy = `If you’ve saved your payment information with Ekõ, your donation will go through immediately:`;
```

Comment:
@subbiahsn It should be `ō`instead of `õ`
```suggestion
export const oneClickCopy = `If you’ve saved your payment information with Ekō, your donation will go through immediately:`;
```